radicle-cli/src/commands/clone.rs

```diff
@@ -175,31 +175,36 @@ pub fn clone<G: Signer>(
 
     // Get seeds. This consults the local routing table only.
     let mut seeds = node.seeds(id)?;
-    if !seeds.has_connections() {
-        return Err(CloneError::NotFound(id));
-    }
-    // Fetch from all seeds.
-    for seed in seeds.connected() {
-        let spinner = term::spinner(format!(
-            "Fetching {} from {}..",
-            term::format::tertiary(id),
-            term::format::tertiary(term::format::node(seed))
-        ));
-
-        // TODO: If none of them succeeds, output an error. Otherwise tell the caller
-        // how many succeeded.
-        match node.fetch(id, *seed)? {
-            FetchResult::Success { .. } => {
-                spinner.finish();
-            }
-            FetchResult::Failed { reason } => {
-                spinner.error(reason);
+    if seeds.has_connections() {
+        // Fetch from all seeds.
+        for seed in seeds.connected() {
+            let spinner = term::spinner(format!(
+                "Fetching {} from {}..",
+                term::format::tertiary(id),
+                term::format::tertiary(term::format::node(seed))
+            ));
+
+            // TODO: If none of them succeeds, output an error. Otherwise tell the caller
+            // how many succeeded.
+            match node.fetch(id, *seed)? {
+                FetchResult::Success { .. } => {
+                    spinner.finish();
+                }
+                FetchResult::Failed { reason } => {
+                    spinner.error(reason);
+                }
             }
         }
+        // TODO: Warn if no seeds were found, and we might be checking out stale data.
```

Comment:
might also be helpful to assign someone to TODOs in comments as well, e.g. `TODO(dave)`

Reply:
Yeah we can start doing that.

Reply:
works well with `grep`, and even ci/bots in future.